model.py

```diff
@@ -3626,7 +3645,7 @@ def calculate_opds_entries(self, verbose=True):
         )
 
 
-    def update_external_index(self, client):
+    def update_external_index(self, client, add_coverage_record=True):
```

Comment:
The SearchIndexCoverageProvider (found in circulation) is going to add a coverage record for every book it indexes, so there's no need to also do the work here.